rebar.config

```diff
@@ -6,11 +6,10 @@
 
 {deps, [
     {clique, ".*", {git, "https://github.com/helium/clique.git", {branch, "develop"}}},
-    {lager, "3.8.1"},
     {observer_cli, "1.6.0"},
-    {jsx, "3.0.0"},
+    {jsx, {git, "https://github.com/lthiery/jsx", {branch, "main"}}},
     {kvc, {git, "https://github.com/etrepum/kvc", {tag, "v1.7.0"}}},
-    {blockchain, {git, "https://github.com/helium/blockchain-core.git", {branch, "master"}}},
+    {blockchain, {git, "https://github.com/helium/blockchain-core.git", {tag, "2021.09.16.1"}}},
```

Comment:
Is this intentional?

Reply:
Hmm, yea I guess I could/should revert without further discussion. What's best practice here? It seems to be that explicit tagging would be favorable?

Reply:
I reverted to branch "master"